unit-tests/src/test/java/org/eclipse/collections/impl/block/procedure/ProceduresTest.java

```diff
@@ -42,6 +44,16 @@ public void throwing()
                 () -> Procedures.throwing(a -> { throw new IOException(); }).value(null));
     }
 
+    @Test
+    public void throwingWithSuccessfulCompletion()
+    {
+        StringBuilder builder = new StringBuilder("a");
+        Procedures.throwing(
+                (ThrowingProcedure<StringBuilder>) stringBuilder -> stringBuilder.append("Visited"))
```

Comment:
Necessary cast?

Reply:
Yes.
Removing it causes
Compilation failure
/unit-tests/src/test/java/org/eclipse/collections/impl/block/procedure/ProceduresTest.java:[52,47] error: cannot find symbol
  symbol:   method append(String)
  location: variable stringBuilder of type Object